src/lib.rs

```diff
@@ -28,6 +28,7 @@ pub mod ebpf;
 pub mod helpers;
 mod jit;
 mod verifier;
+pub mod rust_api;
```

Comment:
Hmm I don't know. I don't like `code_build` much, we're not building “code”. I would accept `insn_build` I think. We can keep searching for a name until we merge. Should ideally be short, for people that do not include `<module>::*` but will prefer to call `<module>::<function>()` a lot.